### DISCUSSION TITLE
Only Industry Factors in csv

### DISCUSSION THREAD

**Discussion by jason-frederick** [USEPA/supply-chain-factors#88] [Q&A] [answered]
Could anyone please post .csv files that contain both the industry and commodity factors? The files currently posted at https://catalog.data.gov/dataset/supply-chain-greenhouse-gas-emission-factors-v1-3-by-naics-6 contain only the industry factors. When I generate the csv files myself using R Studio, they also contain only the industry factors, despite changing the yml files at line 10 to contain: CommodityorIndustryType: "Commodity".

**Comment by bl-young** [comment]
To clarify, whether the factors are `Industry` or `Commodity` is based entirely on that line in the yml file `CommodityorIndustryType`. If you change that to `Industry` it will produce industry factors. The names of sectors are based on NAICS, and will not change to look more like commodity or industry terms.

v1.3 was a commodity model.

**Reply by bl-young** [reply, marked as answer]
So if you can successfully knit the file in R Studio, try changing that line to `Industry` and the numbers should be different (the names will not change).
